gapis/replay/executor/executor.go

```diff
@@ -48,12 +51,14 @@ func Execute(
 	payload protocol.Payload,
 	decoder builder.ResponseDecoder,
 	connection io.ReadWriteCloser,
-	memoryLayout *device.MemoryLayout) error {
+	memoryLayout *device.MemoryLayout,
```

Comment:
Just use the `device.Configuration`, it contains both the `MemoryLayout` and `OS`.

Reply:
device.Configuration will give us the memoryLayout for all ABIs this device supports, but we need the specific one that the capture was taken with.